### PR TITLE
Release notes

### LINKED ISSUE #525: Release notes
Create a living release notes document to let users know what has been included in each major Carina release (e.g. when we do a release with a new version of Swarm and/or Docker).

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #525

### CODE REVIEW

**Comment by everett-toews** [comment]
To merge on the next release. Naturally I'd replace the "XX" in the file before merging.

**Comment by everett-toews** [comment]
Oops. Looks like Jekyll doesn't care for "XX" in the date.

**Comment by everett-toews** [comment]
@jnoller @simonjj @mendeni @thomasem @rgbkrk Have a look if you like.

File: _getting-started/010-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+topics:
+  - docker
+---
+
+### November XX, 2015
+
+#### What’s new
+
+* Enable bind mounting `/var/run/docker.sock` from host to container. e.g. `docker run -v /var/run/docker.sock:/tmp/docker.sock`
+
```
Comment (thomasem) [inline _getting-started/010-release-notes.md]: Should we remove this until we actually release this feature, or is this just where we'd place expected updates that haven't gone out yet?

File: _getting-started/010-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### November 13, 2015
+
+#### What’s new
+
+* [Docker Engine 1.9.0](https://github.com/docker/docker/releases/tag/v1.9.0)
+* [Docker Swarm 1.0.0](https://github.com/docker/swarm/releases/tag/v1.0.0)
```
Comment (thomasem) [inline _getting-started/010-release-notes.md]: What about 2FA support?

Comment (everett-toews) [inline _getting-started/010-release-notes.md]: Added.

File: _getting-started/010-release-notes.md (hunk 1 above)
Comment (everett-toews) [inline _getting-started/010-release-notes.md]: https://github.com/getcarina/getcarina.com/pull/551#issuecomment-157490342

I'd prefer to hold off on merging this until the next release anyway. Pretty excited about `/var/run/docker.sock`!

Comment (thomasem) [inline _getting-started/010-release-notes.md]: Fair enough!

**Comment by rgbkrk** [comment]
This is looking really great! Thanks for starting this up!

File: _getting-started/010-release-notes.md (hunk 1 above)
Comment (thomasem) [inline _getting-started/010-release-notes.md]: I just deployed this feature.

**Pull request merged**